assets/app/styles/_log.less

```diff
@@ -76,6 +77,10 @@
   }
 }
 
+.chromeless .log-scroll-top.affix {
+  right: 0px; // override
+}
```

Comment:
Is `.log-chromeless` still defined? We should remove that and carry over the style that adds margin to the log length warning.